test/extended/builds/new_app.go

```diff
@@ -54,7 +54,7 @@ var _ = g.Describe("[sig-builds][Feature:Builds] oc new-app", func() {
 
 		g.It("should succeed with a --name of 58 characters [apigroup:build.openshift.io]", func() {
 			g.By("calling oc new-app")
-			err := oc.Run("new-app").Args("registry.redhat.io/ubi8/nodejs-16:latest~https://github.com/sclorg/nodejs-ex", "--name", a58, "--build-env=BUILD_LOGLEVEL=5").Execute()
+			err := oc.Run("new-app").Args("quay.io/app-sre/ubi8-nodejs-16:latest~https://github.com/sclorg/nodejs-ex", "--name", a58, "--build-env=BUILD_LOGLEVEL=5").Execute()
```

Comment:
this image definitely exists in registry.redhat.io, are you saying it doesn't behave the same way as the quay.io one?